CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # _seqr_ Changes
 
 ## dev
+* Support OMIM entries with no associated gene and remove phenotypic_series_number (REQUIRES DB MIGRATION)
```

Comment:
ah should I update this for every PR I create against dev?

Reply:
no, generally we only update this with breaking changes for local seqr installations